docs/index.rst

```diff
@@ -15,6 +15,6 @@ Documentation content:
    references/keywords
    roadmap
    setuptools
-   Development guide <developmen/index>
+   Development guide <development/index>
```

Comment:
Looks like because RTD doesn't use stict mode, they've regenerated the docs site with this TOC item simply missing.

Reply:
This should be resolved by https://github.com/pypa/setuptools/pull/2433/files#diff-cde814ef2f549dc093f5b8fc533b7e8f47e7b32a8081e0760e57d5c25a1139d9R13